src/verilog/parser.y

```diff
@@ -157,6 +157,25 @@ static void add_as_subtype(typet &dest, typet &what)
 
 /*******************************************************************\
 
+Function: add_attributes
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+static void add_attributes(YYSTYPE &dest, YYSTYPE &attributes)
+{
+  PRECONDITION(stack_expr(attributes).id() == ID_verilog_attributes);
+  if(!stack_expr(attributes).get_sub().empty())
+    addswap(dest, ID_verilog_attributes, attributes);
```

Comment:
Would it perhaps be better to use `ID_C_verilog_attributes`? They seem to be ignored during type checking at this point, so I am guessing they don't affect the semantics?

Reply:
I don't believe anything with attributes ever gets compared for equality, so it wouldn't make any difference.

The attributes seem to be entirely tool-specific, so there may well be some that have/alter semantics.

I am planning to use them to configure reasoning engines per assertion, but it's not obvious that this is the best mechanism.